### DISCUSSION TITLE
How to use with Inertia/Vue?

### DISCUSSION THREAD

**Discussion by IronSinew** [spatie/laravel-login-link#7] [Q&A] [answered]
We're using Inertia in our project and I'm wondering if there are any plans to allow for a vue component that can be rendered in the same way as the blade component and dropped into an existing vue component?

**Comment by freekmurze** [comment, marked as answer]
We have no plans to create such a component, but in the readme you'll find some instructions on how to create one yourself.

**Reply by IronSinew** [reply]
Oh wow, I'm not sure how I missed that. I read the entire readme on it but did not see the vue implementation section. My apologies. Thanks for answering my question caused by my own miss.
